iceoryx_posh/test/integrationtests/test_publisher_subscriber_communication.cpp

```diff
@@ -285,3 +297,171 @@ TEST_F(PublisherSubscriberCommunication_test, SendingComplexDataType_variant)
                      .has_error());
 }
 
+
+TEST_F(PublisherSubscriberCommunication_test, PublisherBlocksWhenBlockingActivatedOnBothSidesAndSubscriberQueueIsFull)
+{
+    auto publisher = createPublisher<string<128>>(SubscriberTooSlowPolicy::WAIT_FOR_SUBSCRIBER);
+    this->InterOpWait();
+
+    auto subscriber = createSubscriber<string<128>>(QueueFullPolicy::BLOCK_PUBLISHER, 2U);
+    this->InterOpWait();
+
+    EXPECT_FALSE(publisher->publishCopyOf("start your day with a smile").has_error());
+    EXPECT_FALSE(publisher->publishCopyOf("and hypnotoad will smile back").has_error());
```

Comment:
many magic strings in this file...

Reply:
because its hypnotoad and the toad is magical.

I could define all that stuff in multiple variables but this would not improve the code here. We would just add more lines of code.